api.js

```diff
@@ -69,9 +69,10 @@ const handleRequest = (req, res) => { // handles all requests, routing and metho
         }
 
 
-        const student = { ...students[studentIndex], ...req.body } // updates the student information
-
-        return response({ data: student, code: 200 }) // return response
+        const student = { ...students[studentIndex],id: Math.floor(Math.random() * 10).toString()} //, updates the student information
```

Comment:
This only changes the id of the student. What if body of the request contains other details?.